### DISCUSSION TITLE
How to “run `nativemessaging-install install [browser]`”?

### DISCUSSION THREAD

**Discussion by hamaryns** [roelderickx/connective-plugin-linux#29] [Q&A]
I could follow the installation instructions fine until the last step: ran nativemessaging-install.  I cannot find that script anywhere.  Nor seems my command line to know it.  I would appreciate a pointer.

TIA, H.

**Comment by hamaryns** [comment]
OK, forget it, it seems like something went wrong with the pip installation earlier.  I repeated it and now it worked.

Itsme however still does not work since:

> Er is een nieuwe versie van de Connective-plug-in beschikbaar. Klik op de onderstaande upgradeknop en ga verder met de upgradewizard.

**Reply by roelderickx** [reply]
You can try to update the version in [the backend script](https://github.com/roelderickx/connective-plugin-linux/blob/60b33016b1cd4084cfbacdc2a8f3072793d5ba90/connective-backend.py#L731), but I assume the protocol is modified by connective as well.
I am not sure if I am willing to investigate, I'd rather archive this repository. Itsme isn't necessary anyway, it is a solution to a problem which never existed.
